clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/CommonServerStoreProxy.java

```diff
@@ -68,21 +70,21 @@ public void onResponse(EhcacheEntityResponse.ClientInvalidateHash response) {
         final long key = response.getKey();
         final int invalidationId = response.getInvalidationId();
 
-        if (cacheId.equals(messageFactory.getCacheId())) {
+        if (cacheId.equals(cacheId)) {
```

Comment:
Made me realise that is yet another place where that `cacheId` in the message is useless.